src/sidebar/util/dom.js

```diff
@@ -21,3 +21,29 @@ export function listen(element, events, listener, { useCapture = false } = {}) {
     );
   };
 }
+
+/**
+ * Obtain the pixel height of the element with id `elementId`, including
+ * top and bottom margins.
+ *
+ * @param {string} elementId - The DOM element's id attribute
+ * @return {number|null} - The element's height in pixels or `null` if no
+ *                         element with id `elementId` exists
+ */
+export function getElementHeightWithMargins(elementId) {
+  const threadElement = document.getElementById(elementId);
```

Comment:
The diff of this module is out of date with respect to the code in the new thread list implementation. The thread list passes in an `Element`, but this code is still expecting an ID.